pkgs/desktops/budgie/budgie-desktop/default.nix

```diff
@@ -0,0 +1,99 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, accountsservice
+, alsa-lib
+, budgie-screensaver
+, docbook-xsl-nons
+, glib
+, gnome
+, gnome-desktop
+, gnome-menus
+, graphene
+, gst_all_1
+, gtk-doc
+, gtk3
+, ibus
+, intltool
+, libcanberra-gtk3
+, libgee
+, libGL
+, libnotify
+, libpeas
+, libpulseaudio
+, libuuid
+, libwnck
+, mesa
+, meson
+, ninja
+, pkg-config
+, polkit
+, sassc
+, upower
+, vala
+, wrapGAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "budgie-desktop";
+  version = "10.7.1";
+
+  src = fetchFromGitHub {
+    owner = "BuddiesOfBudgie";
+    repo = pname;
+    rev = "v${version}";
+    fetchSubmodules = true;
+    sha256 = "sha256-ww65J9plixbxFza6xCfaz1WYtT9giKkLVH1XYxH41+0=";
+  };
+
+  nativeBuildInputs = [
+    docbook-xsl-nons
+    gtk-doc
+    intltool
+    meson
+    ninja
+    pkg-config
+    vala
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
+    accountsservice
+    alsa-lib
+    budgie-screensaver
+    glib
+    gnome-desktop
+    gnome-menus
+    gnome.gnome-bluetooth_1_0
+    gnome.gnome-settings-daemon
+    gnome.mutter
+    graphene
+    gst_all_1.gstreamer
```

Comment:
Budgie Screenshot app also crashes:

<details>
<summary>Logs:</summary>
<pre>
machine # [   72.508852] gnome-session[1031]: gnome-session-binary[1031]: WARNING: Application 'org.buddiesofbudgie.BudgieDaemon.desktop' killed by signal 5
machine # [   83.987734] .budgie-daemon-[1516]: screenshot.vala:293: Error: no element "playbin"
machine # [   84.356320] traps: .budgie-daemon-[1516] trap int3 ip:7f0bc8e54657 sp:7ffd65c9f820 error:0 in libglib-2.0.so.0.7400.3[7f0bc8e16000+8d000]
machine # [   84.071853] systemd[1]: Started Process Core Dump (PID 1563/UID 0).
machine # [   84.824477] systemd-coredump[1564]: Process 1516 (.budgie-daemon-) of user 1000 dumped core.
machine # 
machine # Module librsvg-2.so.2 without build-id.
machine # Module libpixbufloader-svg.so without build-id.
machine # Module libdconfsettings.so without build-id.
machine # Module libcap.so.2 without build-id.
machine # Module libstdc++.so.6 without build-id.
machine # Module libicudata.so.72 without build-id.
machine # Module libsystemd.so.0 without build-id.
machine # Module libbrotlicommon.so.1 without build-id.
machine # Module libgirepository-1.0.so.1 without build-id.
machine # Module libzstd.so.1 without build-id.
machine # Module liblzma.so.5 without build-id.
machine # Module libgcc_s.so.1 without build-id.
machine # Module libX11-xcb.so.1 without build-id.
machine # Module libxcb-util.so.1 without build-id.
machine # Module libpcre.so.1 without build-id.
machine # Module libGLX.so.0 without build-id.
machine # Module libXdmcp.so.6 without build-id.
machine # Module libXau.so.6 without build-id.
machine # Module libGLdispatch.so.0 without build-id.
machine # Module libdatrie.so.1 without build-id.
machine # Module libxml2.so.2 without build-id.
machine # Module libjson-glib-1.0.so.0 without build-id.
machine # Module libicui18n.so.72 without build-id.
machine # Module libicuuc.so.72 without build-id.
machine # Module libdbus-1.so.3 without build-id.
machine # Module libatspi.so.0 without build-id.
machine # Module libexpat.so.1 without build-id.
machine # Module libbrotlidec.so.1 without build-id.
machine # Module libbz2.so.1 without build-id.
machine # Module libgraphite2.so.3 without build-id.
machine # Module libogg.so.0 without build-id.
machine # Module libvorbis.so.0 without build-id.
machine # Module libpeas-1.0.so.0 without build-id.
machine # Module libunwind.so.8 without build-id.
machine # Module libXRes.so.1 without build-id.
machine # Module libstartup-notification-1.so.0 without build-id.
machine # Module libpcre2-8.so.0 without build-id.
machine # Module libffi.so.8 without build-id.
machine # Module libselinux.so.1 without build-id.
machine # Module libjpeg.so.62 without build-id.
machine # Module libGL.so.1 without build-id.
machine # Module libXrender.so.1 without build-id.
machine # Module libxcb-render.so.0 without build-id.
machine # Module libxcb.so.1 without build-id.
machine # Module libxcb-shm.so.0 without build-id.
machine # Module libpng16.so.16 without build-id.
machine # Module libEGL.so.1 without build-id.
machine # Module libfreetype.so.6 without build-id.
machine # Module libthai.so.0 without build-id.
machine # Module libXinerama.so.1 without build-id.
machine # Module libXrandr.so.2 without build-id.
machine # Module libXcomposite.so.1 without build-id.
machine # Module libXdamage.so.1 without build-id.
machine # Module libXcursor.so.1 without build-id.
machine # Module libXext.so.6 without build-id.
machine # Module libxkbcommon.so.0 without build-id.
machine # Module libXfixes.so.3 without build-id.
machine # Module libtracker-sparql-3.0.so.0 without build-id.
machine # Module libatk-bridge-2.0.so.0 without build-id.
machine # Module libXi.so.6 without build-id.
machine # Module libepoxy.so.0 without build-id.
machine # Module libfribidi.so.0 without build-id.
machine # Module libfontconfig.so.1 without build-id.
machine # Module libpangoft2-1.0.so.0 without build-id.
machine # Module libX11.so.6 without build-id.
machine # Module libharfbuzz.so.0 without build-id.
machine # Module libpangocairo-1.0.so.0 without build-id.
machine # Module libz.so.1 without build-id.
machine # Module libatk-1.0.so.0 without build-id.
machine # Module libltdl.so.7 without build-id.
machine # Module libvorbisfile.so.3 without build-id.
machine # Module libbudgie-plugin.so.0 without build-id.
machine # Module libgstreamer-1.0.so.0 without build-id.
machine # Module libwnck-3.so.0 without build-id.
machine # Module libnotify.so.4 without build-id.
machine # Module libpango-1.0.so.0 without build-id.
machine # Module libcanberra-gtk3.so.0 without build-id.
machine # Module libcanberra.so.0 without build-id.
machine # Module libbudgietheme.so.0 without build-id.
machine # Module libbudgie-private.so.0 without build-id.
machine # Module .budgie-daemon-wrapped without build-id.
machine # Stack trace of thread 1516:
machine # #0  0x00007f0bc8e54657 g_log_structured_array (libglib-2.0.so.0 + 0x5f657)
machine # #1  0x00007f0bc8e54a7e g_log_default_handler (libglib-2.0.so.0 + 0x5fa7e)
machine # #2  0x00007f0bc8e54ce7 g_logv (libglib-2.0.so.0 + 0x5fce7)
machine # #3  0x00007f0bc8e54f7f g_log (libglib-2.0.so.0 + 0x5ff7f)
machine # #4  0x0000000000422e62 budgie_scr_make_screenshot_play_shuttersound.constprop.0 (.budgie-daemon-wrapped + 0x22e62)
machine # #5  0x00000000004266fe budgie_scr_make_screenshot_shoot_screen_co.isra.0 (.budgie-daemon-wrapped + 0x266fe)
machine # #6  0x0000000000426862 ____lambda40__gsource_func (.budgie-daemon-wrapped + 0x26862)
machine # #7  0x00007f0bc8e4dffa g_timeout_dispatch (libglib-2.0.so.0 + 0x58ffa)
machine # #8  0x00007f0bc8e4d4df g_main_context_dispatch (libglib-2.0.so.0 + 0x584df)
machine # #9  0x00007f0bc8e4d898 g_main_context_iterate.constprop.0 (libglib-2.0.so.0 + 0x58898)
machine # #10 0x00007f0bc8e4db4f g_main_loop_run (libglib-2.0.so.0 + 0x58b4f)
machine # #11 0x00007f0bc964f075 gtk_main (libgtk-3.so.0 + 0x1f9075)
machine # #12 0x00000000004123ed main (.budgie-daemon-wrapped + 0x123ed)
machine # #13 0x00007f0bc8a6d24e __libc_start_call_main (libc.so.6 + 0x2924e)
machine # #14 0x00007f0bc8a6d309 __libc_start_main@@GLIBC_2.34 (libc.so.6 + 0x29309)
machine # #15 0x0000000000412475 _start (.budgie-daemon-wrapped + 0x12475)
machine # 
machine # Stack trace of thread 1524:
machine # #0  0x00007f0bc8b46759 __poll (libc.so.6 + 0x102759)
machine # #1  0x00007f0bc8e4d80e g_main_context_iterate.constprop.0 (libglib-2.0.so.0 + 0x5880e)
machine # #2  0x00007f0bc8e4d92c g_main_context_iteration (libglib-2.0.so.0 + 0x5892c)
machine # #3  0x00007f0bc8e4d971 glib_worker_main (libglib-2.0.so.0 + 0x58971)
machine # #4  0x00007f0bc8e77e4d g_thread_proxy (libglib-2.0.so.0 + 0x82e4d)
machine # #5  0x00007f0bc8acce86 start_thread (libc.so.6 + 0x88e86)
machine # #6  0x00007f0bc8b53d30 __clone3 (libc.so.6 + 0x10fd30)
machine # 
machine # Stack trace of thread 1525:
machine # #0  0x00007f0bc8b46759 __poll (libc.so.6 + 0x102759)
machine # #1  0x00007f0bc8e4d80e g_main_context_iterate.constprop.0 (libglib-2.0.so.0 + 0x5880e)
machine # #2  0x00007f0bc8e4db4f g_main_loop_run (libglib-2.0.so.0 + 0x58b4f)
machine # #3  0x00007f0bc90b0d46 gdbus_shared_thread_func (libgio-2.0.so.0 + 0x11ed46)
machine # #4  0x00007f0bc8e77e4d g_thread_proxy (libglib-2.0.so.0 + 0x82e4d)
machine # #5  0x00007f0bc8acce86 start_thread (libc.so.6 + 0x88e86)
machine # #6  0x00007f0bc8b53d30 __clone3 (libc.so.6 + 0x10fd30)
machine # 
machine # Stack trace of thread 1526:
machine # #0  0x00007f0bc8b46759 __poll (libc.so.6 + 0x102759)
machine # #1  0x00007f0bc8e4d80e g_main_context_iterate.constprop.0 (libglib-2.0.so.0 + 0x5880e)
machine # #2  0x00007f0bc8e4d92c g_main_context_iteration (libglib-2.0.so.0 + 0x5892c)
machine # #3  0x00007f0bc3a274ad dconf_gdbus_worker_thread (libdconfsettings.so + 0xb4ad)
machine # #4  0x00007f0bc8e77e4d g_thread_proxy (libglib-2.0.so.0 + 0x82e4d)
machine # #5  0x00007f0bc8acce86 start_thread (libc.so.6 + 0x88e86)
machine # #6  0x00007f0bc8b53d30 __clone3 (libc.so.6 + 0x10fd30)
machine # 
machine # Stack trace of thread 1527:
machine # #0  0x00007f0bc8b4bd3d syscall (libc.so.6 + 0x107d3d)
machine # #1  0x00007f0bc8ea2690 g_cond_wait_until (libglib-2.0.so.0 + 0xad690)
machine # #2  0x00007f0bc8e1b8a1 g_async_queue_pop_intern_unlocked (libglib-2.0.so.0 + 0x268a1)
machine # #3  0x00007f0bc8e1be91 g_async_queue_timeout_pop (libglib-2.0.so.0 + 0x26e91)
machine # #4  0x00007f0bc8e786cd g_thread_pool_thread_proxy (libglib-2.0.so.0 + 0x836cd)
machine # #5  0x00007f0bc8e77e4d g_thread_proxy (libglib-2.0.so.0 + 0x82e4d)
machine # #6  0x00007f0bc8acce86 start_thread (libc.so.6 + 0x88e86)
machine # #7  0x00007f0bc8b53d30 __clone3 (libc.so.6 + 0x10fd30)
machine # ELF object binary architecture: AMD x86-64
machine # 
</pre>
</details>

Reply:
```shell
❯ nix-locate playbin                                
gnome.rygel.out                                  20,848 x /nix/store/0jcbd0lcr1i1s00by8j7khd5vzv2s7x1-rygel-0.42.0/lib/rygel-2.8/plugins/librygel-playbin.so
gnome.rygel.out                                     183 r /nix/store/0jcbd0lcr1i1s00by8j7khd5vzv2s7x1-rygel-0.42.0/lib/rygel-2.8/plugins/playbin.plugin
gst_all_1.gstreamermm.dev                        26,598 r /nix/store/1w6h2ymfr0n9gsvpqp7b98pdhyz1sgy7-gstreamermm-1.10.0-dev/include/gstreamermm-1.0/gstreamermm/playbin.h
gst_all_1.gstreamermm.dev                         1,157 r /nix/store/1w6h2ymfr0n9gsvpqp7b98pdhyz1sgy7-gstreamermm-1.10.0-dev/include/gstreamermm-1.0/gstreamermm/private/playbin_p.h
```